ms-campus-kid-school/src/controllers/careers_controller.go

```diff
@@ -0,0 +1,75 @@
+package controllers
+
+import (
+	"ms_campus_kid_school/src/connection"
+	"ms_campus_kid_school/src/utils"
+)
+
+func CreateCareer(career utils.Career) error {
```

Comment:
It is a convention in the project to name the functions by using camelCase
```suggestion
func createCareer(career utils.Career) error {
```

Reply:
I specify in the description of the ps that some functions begins with uppercase, that's because of is a golang practice that if you need to execute a function by a program that is in other package that function MUST begin with uppercase, is not a caprice,is a go rule. In this case this function will be invoked by routes/career_routes.go a package diferent of controllers.